platform/android/java_godot_lib_jni.cpp

```diff
@@ -165,6 +166,8 @@ JNIEXPORT void JNICALL Java_org_godotengine_godot_GodotLib_ondestroy(JNIEnv *env
 }
 
 JNIEXPORT jboolean JNICALL Java_org_godotengine_godot_GodotLib_setup(JNIEnv *env, jclass clazz, jobjectArray p_cmdline, jobject p_godot_tts) {
+	godot_init_profiler();
```

Comment:
This should be invoked in `initialize` to match the pattern used for the other platforms.